src/Microsoft.Extensions.FileSystemGlobbing/Abstractions/DirectoryInfoWrapper.cs

```diff
@@ -61,29 +86,32 @@ public override DirectoryInfoBase GetDirectory(string name)
                 {
                     // This shouldn't happen. The parameter name isn't supposed to contain wild card.
                     throw new InvalidOperationException(
-                        string.Format("More than one sub directories are found under {0} with name {1}.", _directoryInfo.FullName, name));
+                        string.Format("More than one sub directories are found under {0} with name {1}.",
+                            _directoryInfo.FullName, name));
                 }
             }
         }
 
+        /// <inheritdoc />
         public override FileInfoBase GetFile(string name)
-        {
-            return new FileInfoWrapper(new FileInfo(Path.Combine(_directoryInfo.FullName, name)));
-        }
+            => new FileInfoWrapper(new FileInfo(Path.Combine(_directoryInfo.FullName, name)));
 
+        /// <inheritdoc />
         public override string Name
-        {
-            get { return _isParentPath ? ".." : _directoryInfo.Name; }
-        }
+            => _isParentPath
```

Comment:
Multiline lambda properties kinda harder to read. The single line looked better IMO.